### PR TITLE
change dates to YYYY-MM-DD

### PR DESCRIPTION
**Pull request by osvyg**
Resolves https://github.com/lazd/coronadatascraper/issues/48

I am not sure if this is all that it needs to do?

### CODE REVIEW

File: lib/fetch.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ async function fetch(url, type, date) {
       date = process.env['SCRAPE_DATE'];
     }
     else if (!date) {
-      date = datetime.getYYYYMD();
+      date = datetime.getYYYYMMDD();
```
Comment (mark-otaris) [inline lib/fetch.js]: This date is used to retrieve from cache, which uses YYYY-M-D.

Comment (lazd) [reply lib/fetch.js]: I'll go ahead and change the cache to use YYYY-MM-DD after this

**Comment by lazd** [comment]
@osvyg perhaps we should do this after #42, perhaps as part of #50?

File: lib/fetch.js (hunk 1 above)
Comment (ryanblock) [reply lib/fetch.js]: Glad someone took a look at this, it was on my list!

**Comment by qgolsteyn** [comment]
Closing as #50 will cover these changes

**Pull request closed without merging**